Library/Formula/v8.rb

```diff
@@ -24,24 +22,44 @@ class V8 < Formula
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
   depends_on "readline" => :optional
 
+  needs :cxx11
+
   # Update from "DEPS" file in tarball.
   resource "gyp" do
     url "https://chromium.googlesource.com/external/gyp.git",
-        :revision => "fe00999dfaee449d3465a9316778434884da4fa7"
-    version "2010"
+        :revision => "0bb67471bca068996e15b56738fa4824dfa19de0"
+  end
+
+  resource "icu" do
+    url "https://chromium.googlesource.com/chromium/deps/icu.git",
+        :revision => "f8c0e585b0a046d83d72b5d37356cb50d5b2031a"
+  end
+
+  resource "buildtools" do
+    url "https://chromium.googlesource.com/chromium/buildtools.git",
+        :revision => "b0ede9c89f9d5fbe5387d961ad4c0ec665b6c821"
+  end
+
+  resource "clang" do
+    url "https://chromium.googlesource.com/chromium/src/tools/clang.git",
+        :revision => "5bab78c6ced45a71a8e095a09697ca80492e57e1"
   end
 
   resource "gmock" do
-    url "http://googlemock.googlecode.com/svn/trunk", :revision => 501
-    version "501"
+    url "https://chromium.googlesource.com/external/googlemock.git",
+        :revision => "29763965ab52f24565299976b936d1265cb6a271"
   end
 
   resource "gtest" do
-    url "http://googletest.googlecode.com/svn/trunk", :revision => 700
-    version "700"
+    url "https://chromium.googlesource.com/external/googletest.git",
+        :revision => "be1868139ffe0ccd0e8e3b37292b84c821d9c8ad"
   end
 
   def install
+    # Bully GYP into correctly linking with c++11
+    ENV.libcxx
```

Comment:
Is it C++11 or libc++ you want? If it's the former, use `ENV.cxx11`, which will also set the appropriate flags when building with GCC.